src/main/java/org/gridsuite/modification/server/modifications/GenerationDispatch.java

```diff
@@ -53,6 +57,7 @@ public class GenerationDispatch extends AbstractModification {
 
     private final Map<Integer, Double> totalDemand = new HashMap<>();
     private final Map<Integer, Double> remainingPowerImbalance = new HashMap<>();
+    private final Map<Integer, Double> hvdcBalance = new HashMap<>();
```

Comment:
Why dictionnaries ?

Reply:
these data structures are for each synchronous component ...

Reply:
Theses dictionnaries are only used for tests